minerva-university/CourseSwaps
# fixed back-end testing

Description:
I fixed the errors that were arising in the testing framework as some of the imports in the tests were wrong and the endpoints to some tests were not correctly setup. I also updated the readme with the instructions to setup the testing framework in the back-end.

Issue(s) Addressed:
Changes Made:
Change 1: Deleted the old tests folder
Change 2:Created a new test folder(old one seemed not to be recognized by the unittest framework). Not sure why.
...
How to Test:
1. Pull the new changes 
2. Check the contribution readme with instructions on how to run and create new tests.

Checklist:
 I have performed a self-review of my own code.
 I have commented my code, particularly in hard-to-understand areas.
 My changes generate no new warnings.
 I have added tests that prove my fix is effective or that my feature works.
 New and existing unit tests pass locally with my change. 
 
 
 
![Screen Shot 2023-11-06 at 19 50 47 PM](https://github.com/minerva-university/CourseSwaps/assets/96366736/f43dd899-161b-4c10-881e-1103ac3a0918)
